File: kalkulator.py
num1 = input("Enter a number: ")
if num1.isdecimal() == False:
    print("Nummer kan ikke ha decimaler")
    exit()

    
operasjon = input("Enter an operation: ")
if operasjon != "+" and operasjon != "-" and operasjon != "*" and operasjon != "/":
    print("Operasjonen er ikke gyldig")
    exit()

num2 = input("Enter another number: ")
if num2.isdecimal() == False:
    print("Nummer kan ikke ha decimaler")
    exit()


else:
    if operasjon == "+":
        result = int(num1) + int(num2)
        print(result)

    if operasjon == "-":
        result = int(num1) - int(num2)
        print(result)

    if operasjon == "*":
        result = int(num1) * int(num2)
        print(result)

    if operasjon == "/":
        result = int(num1) / int(num2)
        if result > 0:
            print("nummeret blir med desimaltall")
        else:
            print(result)
    

File: kalkulator.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.question("Enter a number: ", (num1) => {
  if (!Number.isInteger(Number(num1))) {
    console.log("Number cannot have decimals");
    process.exit();
  }

  readline.question("Enter an operation: ", (operation) => {
    if (
      operation != "+" &&
      operation != "-" &&
      operation != "*" &&
      operation != "/"
    ) {
      console.log("Operation is not valid");
      process.exit();
    }

    readline.question("Enter another number: ", (num2) => {
      if (!Number.isInteger(Number(num2))) {
        console.log("Number cannot have decimals");
        process.exit();
      }

      let result;
      switch (operation) {
        case "+":
          result = Number(num1) + Number(num2);
          console.log(result);
          break;
        case "-":
          result = Number(num1) - Number(num2);
          console.log(result);
          break;
        case "*":
          result = Number(num1) * Number(num2);
          console.log(result);
          break;
        case "/":
          result = Number(num1) / Number(num2);
          console.log(result);
          break;
      }

      readline.close();
    });
  });
});
